### ISSUE TITLE
Error running on iOS simulator

### ISSUE THREAD

**Issue by jinzishuai** [jinzishuai/tiktok-flutter-clone#3]
```
╭─   ~/src/jinzishuai/tiktok-flutter-clone   master ···································································································  3s  06:37:47
╰─❯ flutter run
Launching lib/main.dart on iPhone 14 in debug mode...
Updating minimum iOS deployment target to 11.0.
Upgrading project.pbxproj
Upgrading AppFrameworkInfo.plist
Upgrading Podfile
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Upgrading Info.plist
Removing script build phase dependency analysis.
Running pod install...                                             430ms
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Finding Podfile changes
      A image_picker_ios
      A path_provider_foundation
      A video_player_avfoundation
      R image_picker
      R path_provider_ios
      R video_player
      - Flutter
      - cloud_firestore
      - firebase_auth
      - firebase_core
      - firebase_storage
      - sqflite
      - video_compress

    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `cloud_firestore` from `.symlinks/plugins/cloud_firestore/ios`
    cloud_firestore: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
    -> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
    firebase_auth: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
    -> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
    firebase_core: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
    -> Fetching podspec for `firebase_storage` from `.symlinks/plugins/firebase_storage/ios`
    firebase_storage: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
    -> Fetching podspec for `image_picker_ios` from `.symlinks/plugins/image_picker_ios/ios`
    -> Fetching podspec for `path_provider_foundation` from `.symlinks/plugins/path_provider_foundation/ios`
    -> Fetching podspec for `sqflite` from `.symlinks/plugins/sqflite/ios`
    -> Fetching podspec for `video_compress` from `.symlinks/plugins/video_compress/ios`
    -> Fetching podspec for `video_player_avfoundation` from `.symlinks/plugins/video_player_avfoundation/ios`

    Resolving dependencies of `Podfile`
      CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only performed in repo update
    [!] CocoaPods could not find compatible versions for pod "video_player_avfoundation":
      In Podfile:
        video_player_avfoundation (from `.symlinks/plugins/video_player_avfoundation/ios`)

    Specs satisfying the `video_player_avfoundation (from `.symlinks/plugins/video_player_avfoundation/ios`)` dependency were found, but they required a higher minimum
    deployment target.

    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:317:in `raise_error_unless_state'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:299:in `block in unwind_for_conflict'
    <internal:kernel>:90:in `tap'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:297:in `unwind_for_conflict'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:682:in `attempt_to_activate'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:254:in `process_topmost_state'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolution.rb:182:in `resolve'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/molinillo-0.8.0/lib/molinillo/resolver.rb:43:in `resolve'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/resolver.rb:94:in `resolve'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer/analyzer.rb:1082:in `block in resolve_dependencies'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/user_interface.rb:64:in `section'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer/analyzer.rb:1080:in `resolve_dependencies'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer/analyzer.rb:125:in `analyze'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer.rb:422:in `analyze'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer.rb:244:in `block in resolve_dependencies'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/user_interface.rb:64:in `section'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer.rb:243:in `resolve_dependencies'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/installer.rb:162:in `install!'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/command/install.rb:52:in `run'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/lib/cocoapods/command.rb:52:in `run'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/gems/cocoapods-1.12.0/bin/pod:55:in `<top (required)>'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/bin/pod:25:in `load'
    /opt/homebrew/Cellar/cocoapods/1.12.0/libexec/bin/pod:25:in `<main>'

Error running pod install
Error launching application on iPhone 14.
```

**Comment by jinzishuai** [comment]
# Fix

* delete `ios/Podfile.lock`
* bump version
![image](https://user-images.githubusercontent.com/1074685/226177305-063e5eca-8799-48e5-a617-ab37afabc407.png)
* `flutter get` will regenerate the rest of the files 

## Remaining issue

The app shows a blank white screen with 

```
╭─   ~/src/jinzishuai/tiktok-flutter-clone   master *1 !2 ·····························································································  5s  07:06:18
╰─❯ flutter run
Launching lib/main.dart on iPhone 14 in debug mode...
Updating minimum iOS deployment target to 11.0.
Upgrading project.pbxproj
Upgrading AppFrameworkInfo.plist
Upgrading Podfile
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Upgrading Info.plist
Removing script build phase dependency analysis.
Running pod install...                                             13.5s
Running Xcode build...                                                  
 └─Compiling, linking and signing...                         5.0s
Xcode build done.                                           137.5s
[VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: [core/duplicate-app] A Firebase App named "[DEFAULT]" already exists
#0      MethodChannelFirebase.initializeApp (package:firebase_core_platform_interface/src/method_channel/method_channel_firebase.dart:134:11)
<asynchronous suspension>
#1      Firebase.initializeApp (package:firebase_core/src/firebase.dart:43:31)
<asynchronous suspension>
#2      main (package:tiktok_tutorial/main.dart:12:3)
<asynchronous suspension>
Syncing files to device iPhone 14...                                68ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

💪 Running with sound null safety 💪

An Observatory debugger and profiler on iPhone 14 is available at: http://127.0.0.1:54835/xTtqkawXeZc=/
The Flutter DevTools debugger and profiler on iPhone 14 is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:54835/xTtqkawXeZc=/

Application finished.
```

**Comment by jinzishuai** [comment]
ref: https://stackoverflow.com/questions/66899145/flutter-unhandled-exception-core-duplicate-app
